src/lib/coda_base/sparse_ledger.ml

```diff
@@ -2,24 +2,17 @@ open Core
 open Import
 open Snark_params.Tick
 
-module Ledger_hash_binable = struct
-  (* Ledger_hash.t not bin_io *)
-  include Ledger_hash.Stable.V1
+include Sparse_ledger_lib.Sparse_ledger.Make (struct
```

Comment:
There's the same issue here: choosing different-versioned arguments to `Sparse_ledger.Make` gives a different result, so that the `include` will yield a different `Sparse_ledger.t`.